tensorflow_privacy/privacy/membership_inference_attack/membership_inference_attack.py

```diff
@@ -170,6 +171,48 @@ def run_attacks(attack_input: AttackInputData,
       privacy_report_metadata=privacy_report_metadata)
 
 
+def run_seq2seq_attack(attack_input: Seq2SeqAttackInputData,
+                       privacy_report_metadata: PrivacyReportMetadata = None,
```

Comment:
FYI: The PrivacyReportMetadata is currently unused. Is this intentional?

Reply:
Oh, I think I missed this. 

I was planning to call `_compute_missing_privacy_report_metadata()` but saw that:
1. it was calculating the accuracy using `logits` and `labels`, but for seq2seq models these are being lazy loaded
2. it was populating `loss_train` and `loss_test`, which are not provided by the user for seq2seq models

I'm actually not sure if I should include PrivacyReportMetadata, any suggestions?

Reply:
It sounds like this will be a change significant enough to deserve its own PR to me :-) 

By the way, why are logits and labels being lazily loaded? Is the order of magnitude so large that we can't just store them in memory?

If so, we'll need to change the way we compute the accuracy and loss and perhaps bypass _compute_missing_privacy_report_metadata(). Or have a separate function for seq2seq models?

I can mark the variable unused for now and we can implement the rest in a followup PR. What do you think?

Reply:
> By the way, why are logits and labels being lazily loaded? Is the order of magnitude so large that we can't just store them in memory?

Each batch of logits has shape `(num_sequences, num_words, vocab_size)`, so if the `vocab_size` is large enough it might be inefficient to store all batches in memory. Similarly, if each batch has a large number of sequences (and a large number of words in each sequence) storing all the labels at once can become inefficient.

> If so, we'll need to change the way we compute the accuracy and loss and perhaps bypass _compute_missing_privacy_report_metadata(). Or have a separate function for seq2seq models?

Hmm, perhaps we can include a flag for seq2seq models and populate the necessary fields using the existing function, or we can create a separate function `_compute_seq2seq_missing_privacy_report_metadata()` - whichever is cleaner.

> I can mark the variable unused for now and we can implement the rest in a follow-up PR. What do you think?

Sounds good!